### PR TITLE
[MTDSA-3410] AMEND BF Losses RAML

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: resources/public/api/conf/1.0/application.raml [hunk 1]
```diff
@@ -1,61 +1,49 @@
...
+        description: National Insurance number, in the format AA999999A.
+        type: string
+        example: TC663795B
+    /brought-forward:  
+      get: !include broughtForwardLosses_list.raml
+      post: !include broughtForwardLosses_create.raml
+      /{broughtForwardLossId}:
+        uriParameters:
+          broughtForwardLossId:
+            description: National Insurance number, in the format AA999999A.
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/application.raml]: Is broughtForwardLossId and NI number are same??

File: resources/public/api/conf/1.0/broughtForwardLosses_create.raml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: Duplicate headers.contentTypeJson... please delete one :)

File: resources/public/api/conf/1.0/broughtForwardLosses_list.raml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  lossType:
+    description: "Type of Loss.  Can be one of the four loss types: <b><ul> <li> self-employment  <li>self-employment-class4  <li>uk-fhl-property   <li>uk-other-property </b>"
+    example: "self-employment"
+    required: false
+responses:
+  200:
+    body:
+      application/json:
+        description: Retrieve all brought forward losses
+        #type: !include schemas/ListAllBFLoss.json
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/broughtForwardLosses_list.raml]: Please delete the commented ones... they are redundant

File: resources/public/api/conf/1.0/broughtForwardLosses_create.raml (hunk 2 above)
Comment (james-work-account) [reply resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: This is your file.

File: resources/public/api/conf/1.0/schemas/ListAllBFLoss.1.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/schemas/ListAllBFLoss.1.json]: Better have an appropriate file name, instead ListAllBFLoss.1.json

File: resources/public/api/conf/1.0/broughtForwardLosses_list.raml (hunk 3 above)
Comment (james-work-account) [reply resources/public/api/conf/1.0/broughtForwardLosses_list.raml]: Again, these are from the boilerplate. Would you like me to remove them as part of my PR or leave them until the individual stories are worked? You're commenting on files that were created before I branched off your branch.

File: resources/public/api/conf/1.0/errors.raml [hunk 5]
```diff
@@ -54,97 +54,107 @@ traits:
...
-            example:
-              description: A previous amount was submitted for this tax year
-              value:
-                code: RULE_DUPLICATE_???
\ No newline at end of file
+            examples:
+              ruleDuplicateLossAmount:
+                description: A previous amount was submitted for this tax year
+                value:
+                  code: RULE_DUPLICATE_???
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/errors.raml]: can you remove these ???

**Pull request merged**